### ISSUE TITLE
[better-sqlite3] Could not locate the bindings file

### ISSUE THREAD

**Issue by matsumoto0912** [oven-sh/bun#7448]
### What version of Bun is running?

1.0.15+b3bdf22eb

### What platform is your computer?

Linux 5.15.133.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

https://github.com/WiseLibs/better-sqlite3
Follow the guide to install better-sqlite3 and run demo code.

### What do you see instead?

```
error: Could not locate the bindings file. Tried:

```

**Comment by huseeiin** [comment]
`cd node_modules/better-sqlite3`
`bun run install`

**Comment by matsumoto0912** [comment]
Thank you for this!

```
cd node_modules/better-sqlite3
bun run install
```

But there are still some errors after this.

```
bun: symbol lookup error: /my-path/node_modules/better-sqlite3/build/Release/better_sqlite3.node: undefined symbol: node_module_register


```

**Comment by xxxhussein** [comment]
this is unfixed. https://github.com/oven-sh/bun/issues/4290

**Comment by huseeiin** [comment]
> Thank you for this!
> 
> ```
> cd node_modules/better-sqlite3
> bun run install
> ```
> 
> But there are still some errors after this.
> 
> ```
> bun: symbol lookup error: /my-path/node_modules/better-sqlite3/build/Release/better_sqlite3.node: undefined symbol: node_module_register
> ```

this is still unfixed. see https://github.com/oven-sh/bun/issues/4290

**Comment by Jarred-Sumner** [comment]
Fixed by #7132
